winch/codegen/src/masm.rs

```diff
@@ -292,7 +292,7 @@ pub enum RoundingMode {
 
 pub(crate) trait MacroAssembler {
     /// The addressing mode.
-    type Address: Copy;
+    type Address: Copy + Debug;
```

Comment:
I added this because it made some of my debugging a little easier and all the address implementations implement `Debug`.